wom/src/main/scala/wdl/WdlWorkflow.scala

```diff
@@ -75,6 +75,14 @@ case class WdlWorkflow(unqualifiedName: String,
   override lazy val womDefinition: ErrorOr[WorkflowDefinition] = WdlWorkflow.womWorkflowDefinition(this)
 
   /**
+    * Also include workflow outputs which are not technically children but should be processed as such
```

Comment:
Could you expand on this?

IIRC, `children` in WDL is a syntax concept, and workflow outputs should therefore be `children` of the workflow block?

Reply:
I agree that they should but they're not and because of that they were not appearing in the list of Wom nodes.
Rather than going down the rabbit hole of why they're not children and fixing that I added them here.